content/docs/user-guide/experiment-management/experiments-overview.md

```diff
@@ -23,8 +24,8 @@ by default either (see `dvc exp push`).
 Note that DVC Experiments require a unique name to identify them. DVC will
 usually auto-generate one by default, such as `exp-bfe64` (based on the
 experiment's hash). A custom name can be set instead, using the `--name`/`-n`
-option of `dvc exp run`. These names can be used to reference experiments in
-other `dvc exp` subcommands.
+option of `dvc exp run`/`dvc exp save`. These names can be used to reference
```

Comment:
Maybe we link them internally from the cmd ref option sections...

I guess we can see `save` as part of `run`? I'm guessing `run` literally runs `save` (API) internally.